setup-env/config/constants.js

```diff
@@ -12,4 +14,8 @@ module.exports = {
     BROWSERSTACK_BUILD_NAME: 'BROWSERSTACK_BUILD_NAME',
     BROWSERSTACK_PROJECT_NAME: 'BROWSERSTACK_PROJECT_NAME',
   },
+
+  BROWSERSTACK_TEMPLATE: {
```

Comment:
Let's call this BROWSERSTACK_INTEGRATIONS

Reply:
done

Reply:
let's take handshake with Shirish on username/accesskey and not 2 step authentication.

Reply:
Done, shared slack thread with you